src/vsphere_cpi/spec/unit/bosh_release/jobs/cpi/templates/cpi.json.erb_spec.rb

```diff
@@ -24,6 +24,7 @@
           'password' => 'vcenter-password',
           'enable_auto_anti_affinity_drs_rules' => true,
           'upgrade_hw_version' => true,
+          'vm_storage_policy_name' => 'VM Encryption Policy',
```

Comment:
rename this policy name to something else just to avoid confusion since we are not supporting encryption